packages/css/src/footer/footer.scss

```diff
@@ -0,0 +1,28 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright (c) 2023 Gemeente Amsterdam
+ */
+
+@import "../../utils/breakpoint";
+
+.amsterdam-footer__top {
+  background-color: var(--amsterdam-footer-top-background-color);
+  padding-block: 2.5rem;
+  padding-inline: 2rem; /* TODO: For now I've set a padding here, this should eventually be handled by PageGrid */
+}
+
+@media screen and (width > $amsterdam-breakpoint) {
+  .amsterdam-footer__top {
+    grid-template-columns: repeat(var(--amsterdam-page-grid-column-count-wide), 1fr);
+  }
+}
+
+.amsterdam-footer__column {
+  display: flex;
+  flex-direction: column;
```

Comment:
The default value for `align-items` is `normal`, which in grid context is similar to `stretch` ([MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/align-items#values)). So we should add `align-items: start` to `.amsterdam-footer__top`.

I still think we must default to grid layout in cases where we don’t use the flex layout algorithm for the items.

Reply:
Hm, why do you prefer `grid` over `flex` here? `grid` implies a two-dimensional layout, which this isn't. If we would need less css by using `grid` I would get it, but that also isn't the case here.

Reply:
As I said, because we don’t use the flex layout algorithm for the items here.

I don’t agree that grid implies a two-dimensional layout.

Let’s agree to disagree, I can accept flex layout.